components/body/desktop.styl

```diff
@@ -0,0 +1,116 @@
+@media (min-device-width 1200px)
+  .main-side-margin
+    margin 0 170px
+
+  .body-content-container
+    padding-top 0px
+
+  #section1.body-section-content, .section-three .body-section-content, .section-four .body-section-content, .body-section-five-content.js-section-content
+    display flex
+    -webkit-flex-direction row
+    flex-direction row
+
+  .body-section-five-content
+    display block
+
+  .section-two .body-section-content
+    display flex
+    flex-direction row-reverse
+
+  .carousel-track__image
+    width 50%
+
+  .carousel-figure-img
+    width 100%
+    height 25vw
+    background-size cover
+    background-position center center
+
+  figure.slide-images
+    display none
+    width 100%
+    margin 0px
+
+  figure.slide-images div
+    width 100%
+    margin-top 0px
+
+  .carousel-track__image .slideshow-is-active
+    display block
+    figcaption
+      font-size 20px
+
+  .section-one .body-section-content div
+    padding-top 0px
+    max-height 566px
+
+  .body-section-content img
+    max-height 400px
+
+  .body-section-content h1.is-desktop, .section-five h1.is-desktop
+    display block
+    padding-top 0px
+    padding-bottom 30px
+    text-align center
+    font-size 24px
```

Comment:
Try to use the helpers that you imported in https://github.com/artsy/continuum/blob/master/components/stylus_lib/typography.styl !